content/communityPage/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Community Page
```

Comment:
`Node.js Community` is a better title. Writing `Page` doesn't make sense.

Moreover, there are missing `authors` and GitHub edit links.